modules/integration/tests-integration/tests-backend/src/test/resources/testng.xml

```diff
@@ -117,7 +117,6 @@
             <class name="org.wso2.identity.integration.test.oauth2.dcrm.api.OAuthDCRMTestCase"/>
             <class name="org.wso2.identity.integration.test.functions.library.mgt.FunctionLibraryManagementTestCase"/>
             <class name="org.wso2.identity.integration.test.oauth2.OAuth2TokenRevocationWithRevokedAccessToken" />
```

Comment:
why we are removing these test cases?

Reply:
Fixed in 010ae9d4a9657caee9ad2ef796c693143c93e6be